driver/InputFlags.cpp

```diff
@@ -125,8 +128,20 @@ void InputFlags::Parse(int argc, char* argv[])
 
             char short_name = FindShortName(long_name);
 
-            MapInputs[short_name].value = args[i + 1];
-            i++;
+            //check layout flag values
+            if ((args[i + 1]) == ("NCHW")||
```

Comment:
Layout flags aren't the only argument parsed by this function. short_name (line 129) should be checked to make sure error checking is for the correct argument.
Also @JehandadKhan @atamazov are there possible layouts outside of this set?

Reply:
Ok. I will work on this check. Thank you. @cderb

Reply:
Ok. I will work on this check. Thank you @cderb

Reply:
@cderb 
> are there possible layouts outside of this set?

Of course, in the future.